handlers.go

```diff
@@ -438,7 +439,14 @@ func (r *oauthProxy) logoutHandler(w http.ResponseWriter, req *http.Request) {
 		encodedSecret := url.QueryEscape(r.config.ClientSecret)
 
 		// step: construct the url for revocation
-		request, err := http.NewRequest(http.MethodPost, revocationURL, bytes.NewBufferString(fmt.Sprintf("refresh_token=%s", identityToken)))
+		request, err := http.NewRequest(
+			http.MethodPost,
+			revocationURL,
+			bytes.NewBufferString(
+				fmt.Sprintf("token=%s", identityToken),
```

Comment:
Could we also provide `token_type_hint=refresh_token` as described here https://tools.ietf.org/html/rfc7009#section-2.1?

Reply:
identityToken can be refresh/access token, but keycloak only supports refresh/offline tokens, IDP according RFC should infer token type also without hint so it is not necessary